app/controllers/projects_controller.rb

```diff
@@ -25,11 +25,12 @@ def edit
   end
 
   def index
-    if Season.current.active?
-      @projects = Project.selected
-    else
-      @projects = Project.in_current_season.where(aasm_state: %w(accepted proposed))
-    end
+    season = Season.find_by(name: params['filter']) || Season.current
+    @projects = if season.current? and !season.active?
```

Comment:
Does it make sense to make this a 'pre-season' method?
And doublecheck: we need `and` instead of `&&` ? (No need for explanation ;-), just checking)

Reply:
Pre-season: there is `Season#transition?` already. Is that what you mean?

`and` vs. `&&`: I never know what to use. I know I will read up on that after I click the "Comment" button. I know I will have forgotten about it tomorrow   ;)

Reply:
`Season#transition` is from october 1 through dec 31, and I think you want jan 1 through `active`. ? 

The way I remember it is: Use `&&` unless you have a very specific goal for `and`.  
http://www.virtuouscode.com/2014/08/26/how-to-use-rubys-english-andor-operators-without-going-nuts/

Reply:
`and`/`or`  for control flow! Currently tattooing this in my knees! 😄   Thanks!